packages/expo-ui/components/Button/index.tsx

```diff
@@ -75,30 +73,42 @@ export type ButtonProps = {
    * Additional styles to apply to the button.
    */
   style?: StyleProp<ViewStyle>;
+  /**
+   * The text to display inside the button.
+   */
+  children: string;
+  /**
+   * Colors for button's core elements.
+   * @platform android
+   */
+  colors?: {
+    containerColor?: string;
+    contentColor?: string;
+  };
```

Comment:
I think we are missing values for `disabledContainerColor` and `disabledContentColor`